python/cuml/test/test_tsvd.py

```diff
@@ -44,17 +47,21 @@ def test_tsvd_fit(datatype, input_type):
         with_sign = False if attr in ['components_'] else True
         assert array_equal(getattr(cutsvd, attr), getattr(sktsvd, attr),
                            0.4, with_sign=with_sign)
+    if stream is not None:
```

Comment:
Don'T we need to sync in any case? Also is the sync not needed before line 48?

Reply:
Done